cypress/e2e/chi-vue/data-table/server-side/data-table-portal.cy.js

```diff
@@ -275,5 +278,37 @@ describe('Server Side Data Table Portal', () => {
           });
       });
     });
+
+    describe('Full server sort', () => {
+      it('Should sort by default when fullServerSort property is not present', () => {
+        cy.get(`[data-cy='data-table-portal-server-side-default-sort'] .${DATA_TABLE_CLASSES.ROW}`)
+            .first()
+            .find(`.${DATA_TABLE_CLASSES.CELL}`)
+            .eq(3)
+            .find(`.${ICON_CLASS}`)
+            .as('sortIcon');
+        hasClassAssertion('@sortIcon', `${ARROW_UP_CLASS}`);
```

Comment:
This can be placed in a separate function and you can pass a couple of params to it